SmartDeviceLink/SDLSendLocation.h

```diff
@@ -65,4 +64,25 @@
  */
 @property (strong, nonatomic) SDLImage *locationImage;
 
+/**
+  * Mode in which the sendLocation request is sent
+  *
+  * Optional
+  */
+@property (strong, nonatomic) SDLDeliveryMode *deliveryMode;
+
+/**
+   * Timestamp in ISO 8601 format
```

Comment:
That was removed from the proposal spec. It's not in ISO 8601 format because ISO 8601 is a string, not a struct.